demo/react-component/react-dom-0.14.6.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
The demo dependencies are all checked in so we can test with a variety of React versions independent of what is specified in `package.json`.